exercises/ocr-numbers/canonical-data.json

```
@@ -68,7 +68,9 @@
           "    "
         ]
       },
-      "expected": {"error": "Number of input columns is not a multiple of three"}
+      "expected": {
+        "error": "Number of input columns is not a multiple of three"
+      }
```

Should this be (similar to the above change):

```suggestion
      "expected": { "error": "Number of input columns is not a multiple of three" }
```

---

that would not be in line with `prettier`. `format-array` has no intention of formatting objects or other non-array items